### DISCUSSION TITLE
How to extract Data from security Onion

### DISCUSSION THREAD

**Discussion by ashokdev7** [Security-Onion-Solutions/securityonion#6853] [Unsupported Versions] [answered]
Hi 
I want to know how can I get access to data that is being displayed on SO web interface .
I want to extract and display that in my own way 
Like where is this  data that is being stored in elastic or kibana and I want forward it to someone else.

**Comment by dougburks** [comment, marked as answer]
Data is stored in Elasticsearch:
https://docs.securityonion.net/en/2.3/elasticsearch.html
